script/foreman-admin

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env ruby
+# -*- encoding: utf-8 -*-
+
+require 'rubygems'
```

Comment:
Yeah, but see below, if packaged in Foreman then we can simply depend on SCL and use 1.9+ on all platforms.